ui/src/views/Tasks/TaskGroup/index.jsx

```diff
@@ -147,6 +151,7 @@ export default class TaskGroup extends Component {
     super(props);
 
     previousCursor = INITIAL_CURSOR;
+    tasks = new Map();
```

Comment:
Why are these module-level global variables, and not instance properties?

Reply:
It's used inside static methods which don't have access to the `this` instance. What I can do is make `tasks` be static so it can be referred to as `TaskGroup.tasks`.

Reply:
I'll see if there is a way to remove the subscription from happening in `static getDerivedStateFromProps`.

Reply:
Done.